llama-index-core/llama_index/core/evaluation/faithfulness.py

```diff
@@ -58,9 +58,47 @@
     "Otherwise answer NO.\n"
 )
 
+LLAMA3_8B_EVAL_TEMPLATE = PromptTemplate(
+    """Please tell if a given piece of information is supported by the context.
+You need to answer with either YES or NO.
+Answer YES if **any part** of the context supports the information, even if most of the context is unrelated.
+Answer NO if the context does not support the information at all.
+Be sure to read all provided context segments carefully before making your decision.
+
```

Comment:
I believe we are unnecessarily introducing space tokens here. Do the metrics change if the prompt format is similar to the DEFAULT_EVAL_PROMPT?